.github/workflows/release.yml

```diff
@@ -119,7 +122,7 @@ jobs:
         shell: bash
         run: |
           if [ "${{ matrix.job.use-cross }}" == "true" ]; then
-            cross build --release --target=${{ matrix.job.target }} --features=${{ matrix.job.features }}
+            RUSTFLAGS=${{ matrix.job.rust_flags }} cross build --release --target=${{ matrix.job.target }} --features=${{ matrix.job.features }}
```

Comment:
We usually add the `RUSTFLAGS` in the `.cargo/config` inside the crate dir. This an example: https://github.com/elixir-nx/explorer/blob/main/native/explorer/.cargo/config

I think it's better to use that file because people building on those plataforms won't have to configure the flags. WDYT?

Reply:
I didn't even notice that! I will make the change :)